python/resdata/grid/faults/fault_block.py

```diff
@@ -143,9 +143,8 @@ def contains_polyline(self, polyline):
         for p in polyline:
             if GeometryTools.pointInPolygon(p, edge_polyline):
                 return True
-        else:
-            edge_polyline.assertClosed()
-            return GeometryTools.polylinesIntersect(edge_polyline, polyline)
+        edge_polyline.assertClosed()
```

Comment:
This is not equivalent I think. The `else` is only run if the `for` loop has no members.